src/utils/internal/getCallFrame.ts

```diff
@@ -2,25 +2,21 @@
  * Return the stack trace frame of a function's invocation.
  */
 export function getCallFrame() {
-  try {
-    const inspectionError = new Error()
-    inspectionError.name = 'Inspection Error'
-    throw inspectionError
-  } catch (error) {
-    const frames: string[] = error.stack.split('\n')
+  // Error.stack is available even in IE 10, no need for nullish coalesce
```

Comment:
I don't think this comment is necessary. At least, I wouldn't reference MDN link.

Reply:
ok, I can remove both comment lines... (probably not today, still catching up with day-job work)

Reply:
No pressure. I'm thankful for your contribution already!

Reply:
now removed